addons/mail/wizard/mail_resend_message_views.xml

```diff
@@ -16,7 +16,9 @@
                             <field name="name" readonly="1"/>
                             <field name="email" attrs="{'readonly': [('parent.partner_readonly', '=', True)]}"/>
                             <field name="message" readonly="1" class="text-wrap"/>
-                            <field name="partner_id" invisible="1"/>
+                            <field name="notification_id" readonly="1" class="oe_link"/>
+                            <button name="action_open_notification" type="object"
+                                string="Open Notification" class="btn-link" groups="base.group_system"/>
```

Comment:
It is really relevant ? Notifications are technical records, not sure what is the gained flow for people, especially it is limited to system ... I feel like we gain few functional feature compared to technical hacks due to opening records in an embedded tree view. If there are some missing data can't we add it in the wizard itself ?

Reply:
Then I would somehow find another way to display this information. As explained above, I don't think we should send people to technical records, especially the resend wizard is available for everyone, and sending them to an implementation detail (notifications) is way to technical according to me.

What would be the minimum required information to display ? Can't we find something nice ?

Reply:
And the underlying idea is: keep technical stuff hidden (notifications), work with functional stuff (wizard) :)